compiler/passes/normalize.cpp

```diff
@@ -276,6 +278,82 @@ static void transformLogicalShortCircuit()
   }
 }
 
+
+//
+// lowerReduceAssign(): lower the reduce= calls, verifying their correct use.
+//
```

Comment:
I'd expect the above comment to be next to lowerReduceAssign. Also, I'd really like the comment to describe what reduce= starts out as and what it is "lowered" into.